### PR TITLE
Added landing page and home page (css)

### PR DESCRIPTION
**Pull request by Shah-Arsalan**
Screenshot (220)
Screenshot (221)

### CODE REVIEW

**Review by shrey27 — APPROVED** [review]
UI looks nice and clean
Code structure is also very good

PR is good to merge

File: src/Components/Input/Input.css [hunk 1]
```diff
@@ -0,0 +1,34 @@
+form.create-note {
+  position: relative;
+  width: 480px;
+  margin: 30px auto 20px auto;
+  background: #fff;
```
Comment (anuj-kumary) [inline src/Components/Input/Input.css:5]: Don't use hardcoded color value use color as variable

File: src/Components/Input/Input.jsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    <>
+      <div>
+        <form className="create-note">
+          {expansion ? <input name="title" placeholder="Title" /> : null}
+          <textarea
+            onClick={appear}
+            id="MyText"
+            name="content"
+            placeholder="Take a note..."
+            rows={expansion ? 4 : 1}
```
Comment (anuj-kumary) [inline src/Components/Input/Input.jsx:24]: Loved how you expanded your textarea box

File: src/Pages/LandingPage/LandingPage.jsx [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+          </h1>
+          <h3>Your personal quick note taking app</h3>
+          <p>Log in to say hi to your new buddy!</p>
+          <button class="primary-button">Log In</button>
+        </div>
+
+        <div class="landing-image">
+          <img
+            src="https://res.cloudinary.com/dg9tbhjxx/image/upload/v1648225981/hero_i6yhx3.png"
+            alt="image"
```
Comment (anuj-kumary) [inline src/Pages/LandingPage/LandingPage.jsx]: Give proper value to `alt` attributes

**Review by anuj-kumary — APPROVED** [review]

**Pull request merged**